solutions/swb-reference/src/SWBStack.ts

```diff
@@ -166,11 +171,38 @@ export class SWBStack extends Stack {
       subnetIds: SUBNET_IDS
     });
 
-    new SWBApplicationLoadBalancer(this, 'SWBApplicationLoadBalancer', {
+    this._createLoadBalancer(swbVpc, apiGwUrl);
+  }
+
+  private _createLoadBalancer(swbVpc: SWBVpc, apiGwUrl: string): void {
+    const alb = new SWBApplicationLoadBalancer(this, 'SWBApplicationLoadBalancer', {
       vpc: swbVpc.vpc,
       subnets: swbVpc.subnetSelection,
       internetFacing: true // TODO: See if this is required if we are directly passing in subnets
     });
+
+    const proxyLambda = new Function(this, 'LambdaProxy', {
+      handler: 'proxyHandlerLambda.handler',
+      code: Code.fromAsset(join(__dirname, '../../build/proxyHandler')),
+      runtime: Runtime.NODEJS_14_X,
```

Comment:
Since this is new, you might want to use NODEJS_16_X. Before GA we'll have to switch the other lambda to that version as well.